packages/typespec-go/test/cadlranch/authentication/apikeygroup/zz_apikey_client.go

```diff
@@ -32,12 +32,11 @@ func (client *ApiKeyClient) Invalid(ctx context.Context, options *ApiKeyClientIn
 	if err != nil {
 		return ApiKeyClientInvalidResponse{}, err
 	}
-	if !runtime.HasStatusCode(httpResp, http.StatusNoContent, http.StatusForbidden) {
```

Comment:
This is due to the updated TCGC which removed error cases from the non-error responses.